holmes/plugins/prompts/generic_ask.jinja2

```diff
@@ -6,6 +6,9 @@ If you have a good and concrete suggestion for how the user can fix something, t
 
 Use conversation history to maintain continuity when appropriate, ensuring efficiency in your responses.
 
+If you are unsure about the answer to the user's request or how to satisfy their request, you should gather more information. This can be done by asking the user for more information.
```

Comment:
> This can be done by asking the user for more information.

How will holmes ask for more information? and if the user first ask check the status of my pod, and when asked which pod by holmes, the user then input "pod xx in namespace yy", will holmes relates these two inputs?